services/QuillLMS/client/app/bundles/Evidence/components/studentView/container.tsx

```diff
@@ -631,18 +732,18 @@ export const StudentViewContainer = ({ dispatch, session, isTurk, location, acti
     return ''
   }
 
-  function transformMarkTags(node) {
+  const transformMarkTags = React.useCallback((node: any) => {
```

Comment:
Since this is already a really big file that's only getting bigger with these changes, and it looks like this function is only used inside of ReadPassageContainer (and I think is also the only place we were previously using ReactHTMLParser), I'm wondering if it might work to pull this whole function down to a lower-level component, and maybe see if we can get away without as many updates here?